### PR TITLE
Register new package TableTraits.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [davidanthoff/TableTraits.jl](https://github.com/davidanthoff/TableTraits.jl)
Release: [v0.0.1](https://github.com/davidanthoff/TableTraits.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/davidanthoff/TableTraits.jl/builds/265723229)
cc: @davidanthoff

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

@davidanthoff This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by davidanthoff** [comment]
This is the first step in getting rid of a) type piracy in IterableTables.jl and b) requiring Requires.jl.

File: TableTraits/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.6
+NamedTuples 4.0.0
```
Comment (tkelman) [inline TableTraits/versions/0.0.1/requires]: this seems unnecessarily tight, given how little of NamedTuples is being directly used here

Comment (davidanthoff) [inline TableTraits/versions/0.0.1/requires]: Done.

**Comment by davidanthoff** [comment]
Thanks!

File: TableTraits/versions/0.0.1/requires (hunk 1 above)
Comment (tkelman) [reply TableTraits/versions/0.0.1/requires]: same basic comment on TableTraitsUtils by the way

**Pull request merged**